### PR TITLE
Drink resource

### LINKED ISSUE #2: Create basic CRUD endpoints and DTOs for the Drink entity
- [ ] Create `drink.entity.ts`, `create-drink.dto.ts` and `update-drink.dto.ts` with class-validator
- [ ] Create basic crud endpoints for drink entity that calls the drink service
- [ ] Generate the drink service

### LINKED ISSUE #3: Create drink service
Import the prisma service and implement the basic crud operations with it on the drink entity. Pay attention to error handling and be sure to test it (at least) manually!

### PR DESCRIPTION
**Pull request by Tschonti**
Drink DTOs, controller and service

### CODE REVIEW

File: apps/backend/src/drinks/entities/drink.entity.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  @Min(0)
+  alcoholContent: number;
+
+  /**
+   * Whether it was created by a user
+   */
+  @IsBoolean()
+  custom: boolean;
+
+  @IsString()
```
Comment (LeventeFarkashazi) [inline apps/backend/src/drinks/entities/drink.entity.ts:44]: I think we could add examples for the optional fields too.

File: apps/backend/src/drinks/entities/drink.entity.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import { IsBoolean, IsDateString, IsEnum, IsNotEmpty, IsOptional, IsString, IsUUID, Min } from 'class-validator';
+import { IsFloat } from 'src/util/is-float.validator';
+
+export class Drink {
+  @IsUUID()
+  id: string;
+
+  /**
+   * Name of the drink
+   * @example Soproni meggy
```
Comment (LeventeFarkashazi) [inline apps/backend/src/drinks/entities/drink.entity.ts]: The swagger default values don't work, the example string values should be between apostrophes.
  /**
   * Name of the drink
   * @example 'Soproni meggy'
   */

**Review by LeventeFarkashazi — CHANGES_REQUESTED** [review]
Two minor comment changes, but other than that, it's good to go.

File: apps/backend/src/drinks/entities/drink.entity.ts (hunk 2 above)
Comment (Tschonti) [reply apps/backend/src/drinks/entities/drink.entity.ts]: good catch!

**Review by LeventeFarkashazi — APPROVED** [review]

**Pull request merged**